src/main/java/bio/terra/workspace/app/controller/ControlledGcpResourceApiController.java

```diff
@@ -68,13 +74,26 @@ public ControlledGcpResourceApiController(
             .bucketName(body.getGcsBucket().getName())
             .build();
 
+    // This value is serialized in Stairway, so we avoid making a generic list.
+    ControlledResourceIamRoleList privateRoles =
```

Comment:
Is there a standard pattern for serializing generics like Lists in stairway? Deserializing as `List.class` works, but gives a lot of warnings about type safety. This seemed like a reasonable alternative (if a little heavy on boilerplate), but open to other approaches

Reply:
I have always used List.class, but this seems cleaner to me